README.md

```diff
@@ -0,0 +1,170 @@
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Tengo Wallet <a name="about-project"></a>
+
+A smart wallet web app used for enabling the user to track their transactions within categories of their own selection
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li>Ruby</li>
+    <li>Rails</li>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Test-Driven Development
+- Class implementation with methods
+- Data Preserved in .json files
+- Unit tests created with Rspec
+- Backend PGSQL tables
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+cd my-folder
+git clone https://github.com/MarwanDev/tengo-wallet.git
+
+```
+
+### Install
+
+To run the project, execute the folllowing in a terminal.
+
+```sh
+cd my-project
+gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+  rubocop .
+  rspec .
+```
+### Deployment <a name="deployment"></a>
+
+- [Live](https://tengo-wallet.onrender.com/)
+
+- Demo [video](https://drive.google.com/file/d/1WI1qYL1XYgDiYuDx2j0PlAYZpK6zDIa6/view?usp=sharing)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **MarwanDev**
+
+- GitHub: [MarwanDev](https://github.com/MarwanDev)
+- Twitter: [@marwanEg](https://twitter.com/marwaneg)
+- LinkedIn: [marwan-abdelsattar](https://www.linkedin.com/in/marwan-abdelsattar/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Save to file
+- Styling data adding pages
+- Joined tables
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, leave a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse community and The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) for such a great design
```

Comment:
- [ OPTIONAL ] Great job for adding the acknowledgment but I think you linked it to the wrong link so please link it to the Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella). ⭐🤩 so kindly add it 💯

![image](https://github.com/MarwanDev/Tengo-Wallet/assets/58553711/c30fc708-0c7d-49d0-a593-6f42c622e19d)